### ISSUE TITLE
When importing CSV file using Import-DbaCsv, 1 and 0 cannot be imported to bit columns in SQL Server

### ISSUE THREAD

**Issue by rezanid** [dataplat/dbatools#8409]
### Verified issue does not already exist?

Yes

### What error did you receive?

WARNING: [19:41:22][Import-DbaCsv] Failure | String '1' was not recognized as a valid Boolean.

### Steps to Reproduce

```powershell
(Get-ChildItem "$($RootPath)\EmailMessage_1.csv").FullName | Import-DbaCsv -SqlInstance "." -Database MyDB -Table EmailMessage -Truncate -NoTransaction -Schema dbo
```
I believe it is very common for the boolean data to be represented by 0 and 1 (SQL comes to mind ;) ) so I think this should be considered a bug, unless I'm missing something.

### Are you running the latest release?

Yes

### Other details or mentions

I am trying to import data that is split between several CSV files to a tables in SQL server and boolean data is represented by 0 and 1. 

### What PowerShell host was used when producing this error

PowerShell Core (pwsh.exe)

### PowerShell Host Version

PSVersion                      7.2.4
PSEdition                      Core
GitCommitId                    7.2.4
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

### SQL Server Edition and Build number

Microsoft SQL Server 2019 (RTM-CU15) (KB5008996) - 15.0.4198.2 (X64)  	Jan 12 2022 22:30:08  	Copyright (C) 2019 Microsoft Corporation 	Developer Edition (64-bit) on Windows 10 Enterprise 10.0 <X64> (Build 19042: ) (Hypervisor) 

### .NET Framework Version

.NET 6.0.5

**Comment by andreasjordan** [comment]
We are using the LumenWorks Framework and we don't touch the values but just open the file with `$stream = [System.IO.File]::OpenRead($File)` and pass that to LumenWorks. So from my point of view, we can do nothing about this.

**Comment by rezanid** [comment]
I feared that I will receive a comment like this 🙂 fair enough, I will try to check with that library. Perhaps I can find or even fix it there.

**Comment by rezanid** [comment]
No solution exist at the moment. To be investigated further in the csv library being used.

**Comment by petervandivier** [comment]
Confirmed this is fixed by #9479 

<details>
  <summary>Fix repro </summary>

```powershell
Invoke-DbaQuery `
    -SqlInstance localhost `
    -Database tempdb `
    -Query "create table foo ([bit] bit);"

@"
bit
1
false
"@ ./foo.csv 

Import-DbaCsv `
    -SqlInstance localhost `
    -Database tempdb `
    -Table foo `
    -Path ./foo.csv `
    -SingleColumn

Invoke-DbaQuery `
    -SqlInstance localhost `
    -Database tempdb `
    -Query "select * from foo;"
```

![minimal repro showing Import-DbaCsv supports boolean target columns as of 2.1.30](https://github.com/user-attachments/assets/d8e9b07e-f20b-4867-86cc-5406e9612037)

</details>
